sv_pipeline/genome/load_data_tests.py

```diff
@@ -0,0 +1,279 @@
+                                hl.struct(sample_id='SAMPLE-2', gq=52, num_alt=1, cn=3),
+                                hl.struct(sample_id='SAMPLE-3', gq=19, num_alt=1, cn=3),
+                                hl.struct(sample_id='SAMPLE-4', gq=1, num_alt=0, cn=2),
+                                hl.struct(sample_id='SAMPLE-5', gq=31, num_alt=0, cn=2)])
+VARIANT4 = hl.struct(variantId='INS_chr1_10', contig='1', sc=11, sf=0.007703, sn=1428, start=1643228, end=1643309,
+                     sv_callset_Het=11, sv_callset_Hom=0, gnomad_svs_ID='gnomAD-SV_v2.1_INS_1_47',
+                     gnomad_svs_AF=0.00130899995565414, pos=1643228, filters=NULL_STR_ARRAY, xpos=1001643228, cpx_intervals=NULL_INTERVALS,
+                     xstart=1001643228, xstop=1001643309, svType='INS', transcriptConsequenceTerms=['INS'],
+                     sv_type_detail='ME:SVA', sortedTranscriptConsequences=[
+        hl.struct(gene_symbol='CDK11B', gene_id='ENSG00000284664', predicted_consequence='INTRONIC')],
+                     geneIds={'ENSG00000284664'}, samples_no_call=EMPTY_STR_ARRAY, samples_num_alt_1=['SAMPLE-5'],
+                     samples_num_alt_2=EMPTY_STR_ARRAY, genotypes=[hl.struct(sample_id='SAMPLE-1', gq=999, num_alt=0, cn=hl.null('int')),
+                                                      hl.struct(sample_id='SAMPLE-2', gq=999, num_alt=0, cn=hl.null('int')),
+                                                      hl.struct(sample_id='SAMPLE-3', gq=999, num_alt=0, cn=hl.null('int')),
+                                                      hl.struct(sample_id='SAMPLE-4', gq=999, num_alt=0, cn=hl.null('int')),
+                                                      hl.struct(sample_id='SAMPLE-5', gq=1, num_alt=1, cn=hl.null('int'))])
+
+TEST_GUID = 'test_guid'
+TEST_PASSWORD = 'ExamplePasswd'
+TEST_INPUT_DATASET = 'test_dataset/input_vcf.gz'
+TEST_MT_PATH = 'test_mt/data.mt'
+TEST_GENCODE_PATH = 'test_gtf/gtf'
+TEST_HOST = 'TEST_HOST'
+TEST_PORT = '9500'
+TEST_GENCODE_RELEASE = 31
+TEST_BLOCK_SIZE = 3000
+TEST_NUM_SHARDS = 10
+TEST_INDEX_NAME = 'test_guid__structural_variants__wgs__grch38__20210426'
+
+
+class LoadDataTest(unittest.TestCase):
+    def setUp(self):
+        self.vcf_file = tempfile.mkstemp(suffix='.vcf')[1]
+        with open(self.vcf_file, 'w') as f:
+            f.writelines('\n'.join(VCF_DATA))
+        hl.init()
+        self.mt = hl.import_vcf(self.vcf_file, reference_genome='GRCh38')
+
+    def tearDown(self):
+        hl.stop()
+        os.remove(self.vcf_file)
+
+    @mock.patch('sv_pipeline.genome.load_data.os')
+    @mock.patch('sv_pipeline.genome.load_data.hl')
+    @mock.patch('sv_pipeline.genome.load_data.logger')
+    def test_load_mt(self, mock_logger, mock_hl, mock_os):
+        mock_os.path.splitext.return_value = ['test_dataset/input_vcf', 'gz']
+        mock_os.path.isdir.return_value = True
+        _ = load_mt(TEST_INPUT_DATASET, '', False)
```

Comment:
should also test other permutation of this function - maybe one test with matrixtable_file provided but the file doesnt exist, and one test where the file does exist but overwrite_matrixtable is True